hpx/util/unwrapped.hpp

```diff
@@ -495,20 +507,66 @@ namespace hpx { namespace util
                     util::detail::pack<typename std::decay<Ts>::type...>
                 >::type type;
         };
-    }
-
-    ///////////////////////////////////////////////////////////////////////////
+    } // end namespace detail
```

Comment:
Interesting, I was not aware of that. May I ask to add it anyways for consistency?